int8 DETECTED = 1
int8 IN_GRIPPER = 3
int8 IN_SLOT = 7

int8 status
